Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1624,5 +1638,26 @@ public class TipChangedEventArgs : EventArgs
             /// </summary>
             public HashDigest<SHA256> Hash { get; set; }
         }
+
+        /// <summary>
+        /// Provides data for the <see cref="BlockChain{T}.Reorged"/> event.
+        /// </summary>
+        public class ReorgedEventArgs : EventArgs
+        {
+            /// <summary>
+            /// The <see cref="Block{T}"/> before chain swapped.
+            /// </summary>
+            public Block<T> OldTip { get; set; }
+
+            /// <summary>
+            /// The <see cref="Block{T}"/> after chain swapped.
```

Comment:
I want to hear your opinion about the expression, `The Block<T>`. Because it just describes as `The Block<T>` without the description of the tip, so it is hard to know what the referenced block is among blocks after or before swapped when we looked only the comment. Of course, we can know it from the name of the property, but IMO it will be better to describe that the block is the tip block. If you agree with my opinion, I suggest referencing `BlockChain<T>.Tip` instead of `Block<T>`?

```suggestion
            /// The <see cref="BlockChain{T}.Tip"/> after chain swapped.
```

Reply:
@moreal :+1: Seems much better.

One more thing I want to suggest @riemannulus is that how about adjusting the term “swap” here.  Actually, we've never set the consistent terminology for reog/fork/swap etc yet, I believe it's good chance to fix this and give consistent nuance to these similar terms.  IMHO I feel the word “reorg” is way more appropriate here (i.e., “after the chain is reorged”).  /cc @libplanet

Reply:
@moreal I highly agreed with your opinion.

@dahlia It is a much better way to represent the meaning. I agree with that.

Reply:
change it https://github.com/planetarium/libplanet/pull/945/commits/5f413117f96112ae0336dcf72a461dd5b55209a1